cogs/strike.py

```diff
@@ -183,7 +183,7 @@ async def yes_out_of_sync_ban_member_button_callback(self, _: discord.Button, in
         the manual moderation tracker subroutine that sent the view,
         so all that is required is to delete the original message that sent this view.
         """
-        logger.debug("\"Yes\" button pressed. %s", interaction)
+        logger.debug('"Yes" button pressed. %s', interaction)
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes